tds_login_test.go

```diff
@@ -328,19 +331,20 @@ func TestLoginWithADALManagedIdentityAuth(t *testing.T) {
 		[]string{
 			"  12 01 00 35 00 00 01 00  00 00 1F 00 06 01 00 25\n" +
 				"00 01 02 00 26 00 01 03  00 27 00 04 04 00 2B 00\n" +
-				"01 06 00 2C 00 01 ff 00  00 00 00 00 00 00 00 00\n" +
+				"01 06 00 2C 00 01 ff 00  00 14 00 00 00 00 00 00\n" +
```

Comment:
> 00 14 [](http://example.com/codeflow?start=29&length=6)

i need to change these version bytes to placeholders and fill in the real values at run time